xcode/Subconscious/Shared/Components/Detail/DetailStackView.swift

```diff
@@ -318,12 +374,12 @@ struct DetailStackModel: Hashable, ModelProtocol {
 
         // If slashlink pointing to our sphere, dispatch findAndPushEditDetail
         // to find in local or sphere content and then push editor detail.
-        guard slashlink.peer != nil else {
+        guard address.peer != nil else {
             return update(
                 state: state,
                 action: .findAndPushMemoEditorDetail(
-                    slug: slashlink.toSlug(),
-                    fallback: link.fallback
+                    slug: address.toSlug(),
+                    fallback: ""
```

Comment:
We don't want to lose the fallback here. The fallback is the content that is autopopulated when you create a new note.

When you tap on a link and the note doesn't exist, we auto-populate the note with the link text (not the slug, but the prose text).

Reply:
Refactored to flow the `link` through from `findAndPushLinkDetail` to `findAndPushDetail`.